presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/RemoveUnsupportedDynamicFilters.java

```diff
@@ -123,6 +125,9 @@ public PlanWithConsumedDynamicFilters visitJoin(JoinNode node, Set<String> allow
         {
             JoinDynamicFilterResult joinDynamicFilterResult = extractDynamicFilterFromJoin(node, allowedDynamicFilterIds);
             if (!joinDynamicFilterResult.getProbe().equals(node.getLeft()) || !joinDynamicFilterResult.getBuild().equals(node.getRight()) || !joinDynamicFilterResult.getDynamicFilters().equals(node.getDynamicFilters())) {
+                Optional<RowExpression> filter = node
+                        .getFilter().map(this::removeAllDynamicFilters)  // dynamic filtering is not supported for LookupJoinOperators
+                        .filter(expression -> !expression.equals(TRUE_LITERAL));
```

Comment:
`TRUE_CONSTANT`. I assume this is cherry picked from prestosql? Do you have a reference?

Reply:
https://github.com/prestosql/presto/pull/2659